### PR TITLE
Admin view

### PR DESCRIPTION
**Pull request by andrew-wzj**
Start creating the admin view for the housing lottery.
Missing applicant's standing for application table
You may notice the stand columns are missing from the row since I haven't figured out to get the highest standing from the applicants
It would make another 5 columns to view the current standing of each applicant and find the highest-standing applicant to assign the lottery number

### CODE REVIEW

File: src/views/HousingLottery/adminView/index.jsx [hunk 1]
```diff
@@ -1,23 +1,137 @@
...
+      const applicants = await housingService.getAllApplicants();
+      const schoolYear = await housingService.getAllSchoolYear();
+
+      // Combine the fetched data into a single array of objects
+      const combinedData = applicants.map((applicant, index) => {
+        const rowData = {
+          lotteryNumber: index + 1,
+          preference: JSON.stringify(preference[index]), // Assuming preference is an object
+        };
+
```
Comment (ahwnsals1234) [inline src/views/HousingLottery/adminView/index.jsx:38]: Just for clarification, is the variable 'i' here representing each applicant's email address?

**Review by ahwnsals1234 — COMMENTED** [review]
LGTM!!

**Comment by ahwnsals1234** [comment]
Just a thought, maybe might not be entirely necessary for MVP, but do you think it'd be worth it to add error handling functions as well?

**Review by andrew-wzj — COMMENTED** [review]
ensure the env api links correct

**Pull request merged**